RDKBACCL-1239: Intermittently(> 50 % of times), Network unreachable o…

manigandanj opened the pull request:
…bserved in USB Tethering (RNDIS) Support

Reason for change: Not required to send IP information from CellularManager-MM for RNDIS interface.
Test procedure: Network unreachable issue is not observed. Risks: None

kpandu987 commented:
looks okay to me

arunkumar-palanichamy reviewed (commented):
This is needed for Modem USB dongles. What is the issue sending IP details to WanManager?

arunkumar-palanichamy reviewed (changes requested):
please check.

manigandanj committed:
RDKBACCL-1239: Intermittently(> 50 % of times), Network unreachable observed in USB Tethering (RNDIS) Support

Reason for change: Not required to send IP information from
CellularManager-MM for RNDIS interface.
Test procedure: Network unreachable issue is not observed.
Risks: None

Signed-off-by: Manigandan Gopalakrishnan <Manigandan_Gopalakrishnan@comcast.com>
